app/models/deck.rb

```diff
@@ -8,18 +8,18 @@ def generate_countries_deck
   end
 
   def create_cards(data)
-    data.each do |row|
```

Comment:
Changed 'row' to 'country_hash' to more clearly represent the type of object we are getting from the api.